README.md

```diff
@@ -29,10 +29,10 @@ From within Visual Studio:
 
 1. Open the Solution Explorer.
 2. Right-click on a project within your solution.
-3. Click on *Manage NuGet Packages...*
-4. Click on the *Browse* tab and search for "Stripe.net".
+3. Click on _Manage NuGet Packages..._
+4. Click on the _Browse_ tab and search for "Stripe.net".
 5. Click on the Stripe.net package, select the appropriate version in the
-   right-tab and click *Install*.
+   right-tab and click _Install_.
```

Comment:
Are those changes expected @rmanzer-stripe ?

Reply:
I did not make those changes.  That is not where I was modifying this file.  I'm confused how those changes were introduced.

Reply:
I think it's your config of VSCode that likely caused those changes/replacements

Reply:
yeah please undo those changes and then push again and we can merge!

Reply:
Stars restored.  I will `save without formatting` for all README updates going forward.